atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/OracleDbTableFactory.java

```diff
@@ -95,7 +97,7 @@ public DbWriteTable createWrite(TableReference tableRef, ConnectionSupplier conn
         TableValueStyle tableValueStyle = valueStyleCache.getTableType(conns, tableRef, config.metadataTable());
         switch (tableValueStyle) {
             case OVERFLOW:
-                return OracleOverflowWriteTable.create(config, conns, oracleTableNameGetter, tableRef);
+                return OracleOverflowWriteTable.create(config, conns, oracleTableNameGetter, oraclePrefixedTableNames, tableRef);
```

Comment:
should we just create the `OraclePrefixedTableNames` here rather than storing in field in `OracleDbTableFactory` constructor, or potentially push into the `OracleOverflowWriteTable#create` method itself?